### PR TITLE
Login add popup

### PR DESCRIPTION
**Pull request by SachJay**
adds popup on login 
adds code for popup on logout but it didn't work so I just removed it

### CODE REVIEW

File: client/src/components/utils/Footer/Footer.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Anchor, Footer as SiteFooter, Text } from "grommet";
 export default function Footer() {
   return (
     <SiteFooter
-      style={{ padding: 20, position: "fixed", bottom: 0, width: "100vw" }}
+      style={{ padding: 20, position: "static", bottom: 0, width: "100vw" }}
```
Comment (mbasak99) [inline client/src/components/utils/Footer/Footer.js:7]: Revert this

**Review by mbasak99 — CHANGES_REQUESTED** [review]
Change the file where I commented.

File: client/src/components/utils/Footer/Footer.js (hunk 1 above)
Comment (SachJay) [reply client/src/components/utils/Footer/Footer.js:7]: static is correct, fixed is the foot is always at the bottom of the screen
we don't want that

Comment (mbasak99) [reply client/src/components/utils/Footer/Footer.js:7]: It doesn't look good on my screen

**Review by mbasak99 — CHANGES_REQUESTED** [review]
This shit sucks ass bro.

**Review by mbasak99 — APPROVED** [review]
Cool

**Review by rabiaq123 — APPROVED** [review]

**Pull request merged**